data/ka'het missions.txt

```diff
@@ -93,10 +93,11 @@ mission "First Contact: Ka'het: Remnant 1B"
 
 mission "First Contact: Ka'het: Quarg 1"
 	name "Abandoned Ringworld"
-	description "Visit a Quarg station to ask them about the abandoned ring in the middle of Ka'het space."
+	description "Visit the Quarg and ask them about the abandoned ring in the middle of Ka'het space."
 	landing
 	source "Querii'ka"
-	destination "Lagrange"
+	to complete
+		has "First Contact: Z Ka'het: Quarg 2: done"
```

Comment:
This is an impossible condition to obtain (Ka'het: Quarg 2 is never accepted and therefore can never be done), so you should just use the `never` condition.
```suggestion
		never
```

Reply:
fixed